docs/create-sharded-table.rst

```diff
@@ -0,0 +1,120 @@
+.. _create-sharded-table:
+
+====================
+Create sharded table
+====================
+
+One core concept CrateDB uses to distribute data across a cluster is sharding. 
+CrateDB splits every table into a configured number of shards, which are 
+distributed evenly across the cluster. You can think of shards as sub-tables 
+with corresponding sub-indexes. If we create a table like the following:
```

Comment:
```suggestion
distributed evenly across the cluster. You can think of shards as sub-tables. If we create a table like the following:
```

`index` is unfortunately a bit overloaded. It could be a ES index (which equals a shard), or it could refer to the index structures underneath. So we might want to avoid mentioning it here without a clarification to avoid confusion.

Reply:
Actually, might also want to avoid using the `sub-tables` phrase here, so that it can be used to explain what a partition is.

"A shard is a self-contained part of a table", or something like that?

Reply:
How about

"You can think of shards as a self-contained part of a table, that includes both a subset of records and 
corresponding indexing structures."